parts/chapters/sections/11/2.fodt

```diff
@@ -10613,22 +10613,22 @@ Updated with AFR/TSA Rev-D comments and new keywords.</dc:description><meta:init
        <text:p text:style-name="P235">or just for numerical aquifers one, two and three:</text:p>
        <text:p text:style-name="_40_Example">ANQR</text:p>
        <text:p text:style-name="_40_Example">1 <text:s/>2 <text:s/>3 /</text:p>
-       <text:p text:style-name="_40_Example">ANQR</text:p>
+       <text:p text:style-name="_40_Example">ANQT</text:p>
        <text:p text:style-name="_40_Example">1 <text:s/>2 <text:s/>3 /</text:p>
        <text:p text:style-name="P244"/>
       </table:table-cell>
      </table:table-row>
      <table:table-row table:style-name="Table1793.4">
       <table:covered-table-cell table:style-name="Table1793.A2"/>
       <table:table-cell table:style-name="Table1793.A2" office:value-type="string">
-       <text:p text:style-name="P295">Analytical aquifers that have been allocated to an analytical aquifer list via the AQUALIST keyword in the SOLUTION section are identified by the first two characters AL in the variable mnemonic.</text:p>
```

Comment:
In the same table there is the sentence:
"Analytical aquifers are identified by the first two characters AA in the variable mnemonic and numerical aquifers are identified by the first two characters of AQ."

Shouldn't this be AN instead of AQ?

Reply:
Indeed it should, thanks for spotting it